client/app/routes/collection/add.js

```diff
@@ -1,4 +1,124 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
+
+    model() {
+        return {
+            submission_form_name: 'Preprints Submission Form',
+            sections: [
+                'upload',
+                'disciplines',
+                'basic info',
+                'authors',
+                'submit'
+            ],
+            initial_state: {
+                upload_section: {
+                    state: ['unsaved', 'editing'],
```

Comment:
I think it would be clearer to replace `state` with more specific fields such as  `saveState`, `editState`, `isDefined`, etc.